pheweb/serve/data_access/db.py

```diff
@@ -408,9 +408,9 @@ def get_coding(self):
 
 class MissingVariantDB(object):
     @abc.abstractmethod
-    def get_missing_variant(self, variant: Variant):
-        """Retrieve missing variant data
-        Returns: missing variant results
+    def get_missing_variant(self, variant: Variant) -> Optional[Dict[str,Any]] | None:
```

Comment:
great, just the minor nitpick that `Optional[Dict[str,Any]]` already means the same as `Dict[str,Any]|None`, so the ` | None` in the end is unnecessary.